contrib/datastore/file/gcp/gcp.go

```diff
@@ -30,7 +44,44 @@ type GcpAdapter struct {
 }
 
 func (g GcpAdapter) CreateFileShare(ctx context.Context, fs *file.CreateFileShareRequest) (*file.CreateFileShareResponse, error) {
-	panic("implement me")
+	instanceName := fs.Fileshare.Name + "-instance"
```

Comment:
Why to append "-instance" ? Does GCP expects the name to be suffixed with this?

Reply:
No GCP doesn't expect that suffix, used this approach to get utilization of FileShare name from multi-cloud API itself to  provide GCP backend operation for Filestore Instance creation.
Any other suggestion?

Reply:
Leave the name as provided by user. Please don’t add suffix.

Reply:
GCP Cloud Backend API needs Instance Id and FileShare Name. Just to distinguish instance & fileshare. I used this prefix. I am not taking instance as a separate input from User. Just tried to combine the logic from SODA multi-cloud for GCP backend operation.
Please refer following snapshot.
![image](https://user-images.githubusercontent.com/8712951/93693512-57de5280-fb1e-11ea-9a12-a019f88eddae.png)
![image](https://user-images.githubusercontent.com/8712951/93693524-83613d00-fb1e-11ea-95a5-a521defde216.png)

Reply:
ok